### PR TITLE
docs: Add section on server codegen

### PR DESCRIPTION
**Pull request by oyvindberg**
Hey there. I think we need to showcase the new functionality for server stubs. Thoughts?

### CODE REVIEW

**Review by ghostdogpr — APPROVED** [review]
Looks good! I'll merge it only after the next release as it gets published automatically

**Comment by oyvindberg** [comment]
nice! can we have a release soon? 🙏

**Comment by ghostdogpr** [comment]
Yeah I think once we merge the Caching PR we could do a release. I'm traveling in Europe this week and the next so I may be less responsive than usual but will work on it when I can.

**Pull request merged**
